Source/Core/DolphinWX/Config/AdvancedConfigPane.h

```diff
@@ -5,6 +5,9 @@
 #pragma once
 
 #include <ctime>
+
+#include <wx/dateevt.h>
```

Comment:
Ah, thank you for the recommendation, I wasn't aware of this technique. (I guess I should have noticed the already existing forward declarations in the file.) This appears to be a really good way to avoid unnecessary dependencies. I will change it.

Do you have guidelines for future contributions? What are good candidates for forward declarations? Only classes and structs? Or functions as well?